src/lib/loaf-node.js

```diff
@@ -1,63 +1,67 @@
 "use strict";
 
-const { URL } = require('url');
-const { httpsSend, makeConfigs } = require('../utils/send');
+import { URL } from "url";
+import { put, update, makeConfigs } from "../utils/send.js";
 
 const DEFAULT_CONFIG = {
   test: false,
   debug: false,
 };
 
-function sendEvent(host, path, eventName, userId, eventAttributes) {
-  // TODO; error handling for if `eventName` is missing. `eventName` is required.
-  // TODO; DISCUSS - should userId be required? We can lump unassociated events under a 'default user'
+async function sendEvent(host, path, eventName, userId, eventAttributes) {
+  const eventData = {
+    eventName,
+    userId,
+    eventAttributes,
+  };
 
-  const eventData = JSON.stringify({
-    event_name: eventName,
-    user_id: userId,
-    event_attributes: { ...eventAttributes }
-  });
+  const response = await put(
+    "event",
+    eventData,
+    makeConfigs(host, path, "POST", eventData),
+  );
 
-  httpsSend(eventData, makeConfigs(host, path, "POST", eventData.length));
+  return response;
 }
 
-function makeUser(host, path, userId, userAttributes) {
-  // TODO; error handling for if `userId` is missing. `userId` is required.
+async function makeUser(host, path, userId, userAttributes) {
+  const userData = {
+    userId,
+    userAttributes,
+  };
 
-  const userData = JSON.stringify({
-    user_id: userId,
-    user_attributes: { ...userAttributes },
-  });
+  const response = await put(
+    "user",
+    userData,
+    makeConfigs(host, path, userData),
```

Comment:
The `makeConfigs` function isn't being invoked with the correct arguments according to the new version of the function.